suricata.yaml.in

```diff
@@ -613,12 +613,34 @@ af-packet:
     #  Requires at least Linux 3.14.
     #  * cluster_rollover: kernel rotates between sockets filling each socket before moving
     #  to the next. Requires at least Linux 3.10.
+    #  * cluster_ebpf: kernel rotates between sockets following an extended BPF function loaded
+    #  from a file. User needs to use ebpf-lb-file option to set the path to the file
+    #  to load. Requires at least Linux 4.2.
     # Recommended modes are cluster_flow on most boxes and cluster_cpu or cluster_qm on system
     # with capture card using RSS (require cpu affinity tuning and system irq tuning)
     cluster-type: cluster_flow
     # In some fragmentation case, the hash can not be computed. If "defrag" is set
     # to yes, the kernel will do the needed defragmentation before sending the packets.
     defrag: yes
+    # eBPF file containing a 'loadbalancer' function that will be inserted into the
```

Comment:
I'm inclined to reject this conf update. I think its a too advanced topic to be in the default config.

Reply:
You want to remove the full ebpf section and let use use the doc ?

Reply:
yeah, something like 'for advanced config options for using ebpf and/or xdp see <doc url>'

Reply:
see _docurl_